### PR TITLE
Fix bad display of header toolbar in Product page and Stock page

### LINKED ISSUE #27085: [BO] Bad display of header toolbar in Product page and Stock page
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

An unexpected horizontal line will be displayed at top of the Product edit page and the Stock page.

**Product page:**

Screenshot 2021-12-24 at 13-35-57 Product • dev

**Stock page:**

Screenshot 2021-12-24 at 13-35-18 Stock • dev

### Expected behavior

This horizontal line doesn't appear in other pages.

**Categories page:**

Screenshot 2021-12-24 at 13-43-20 Edit Clothes • dev

### Steps to reproduce

**Product:**
1. Go to BO > Catalog > Products
2. Select a product to edit

**Stock:**
1. Go to BO > Catalog > Stock

### PrestaShop version(s) where the bug happened

8.0

### PHP version(s) where the bug happened

8.0

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by mparvazi**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Hide unexpected lines at the top of Product page and Stock page.
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #27085
| How to test?      | Go to BO > Catalog > Products > select a product<br/>Go to BO > Catalog > Stock<br/>Header toolbar dsipay in these pages must be similar to other pages (Multistore is considered)
| Possible impacts? | UI in Product page (also V2) and Stock page

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27283)

### CODE REVIEW

File: admin-dev/themes/new-theme/scss/components/layout/_content_div.scss [hunk 1]
```diff
@@ -28,9 +28,20 @@
...
     .header-toolbar {
-      padding-left: 0;
-      margin: 0 -#{$grid-gutter-width / 2};
-      margin-bottom: 1rem;
+      display: none;
+    }
+  }
+
+  @at-root .multishop-enabled & {
+
```
Comment (sowbiba) [inline admin-dev/themes/new-theme/scss/components/layout/_content_div.scss]:
```suggestion
```

File: admin-dev/themes/new-theme/scss/components/layout/_content_div.scss [hunk 2]
```diff
@@ -28,9 +28,20 @@
...
+    &.-notoolbar {
+
```
Comment (sowbiba) [inline admin-dev/themes/new-theme/scss/components/layout/_content_div.scss]:
```suggestion
```

File: admin-dev/themes/new-theme/scss/components/layout/_content_div.scss [hunk 3]
```diff
@@ -28,9 +28,20 @@
...
+      .header-toolbar {
+        display: block;
```
Comment (sowbiba) [inline admin-dev/themes/new-theme/scss/components/layout/_content_div.scss:38]: If the class has -notoolbar, then header-toolbar is not displayed => that sounds logical to me.
Why there is an exception for multistore ? or do I understand wrong ?

File: admin-dev/themes/new-theme/scss/components/layout/_content_div.scss (hunk 1 above)
Comment (mparvazi) [reply admin-dev/themes/new-theme/scss/components/layout/_content_div.scss]: These empty lines means there is a parent child relationship.
So debugging would be easier.
Btw these empty lines will be deleted by Sass after compile.

```
.parent {
    color: $white;
    
    .child {
        background: $dark;
        text-align: $align;
    }
}
```

File: admin-dev/themes/new-theme/scss/components/layout/_content_div.scss (hunk 3 above)
Comment (mparvazi) [reply admin-dev/themes/new-theme/scss/components/layout/_content_div.scss:38]: By default `-notoolbar` means that header-toolbar must be hidden. This class used for pages that doesn't require header-toolbar (e.g. Product page).

The multistore-toolbar will be inserted inside `content-div`
```
<div class="content-div -notoolbar ">
    <MultiStoreHeader/>
    <HeaderToolbar/>
    <PageContent/>
</div>
```
Page doesn't need header-toolbar but multistore has header-toolbar class.
So this exception must be handled.

File: admin-dev/themes/new-theme/scss/components/layout/_content_div.scss (hunk 1 above)
Comment (NeOMakinG) [reply admin-dev/themes/new-theme/scss/components/layout/_content_div.scss]: It doesn't help the readability if the parent doesn't have any attribute, the tabulation does it by itself, I agree with @sowbiba on this I never did this, but maybe we could autofix it with stylelint

File: admin-dev/themes/new-theme/scss/pages/product/product_page.scss [hunk 4]
```diff
@@ -112,9 +112,9 @@ $product-page-padding-bottom: 80px !default;
   .product-header {
     padding-top: 1.25rem;
     padding-bottom: 1.25rem;
-    margin-top: -$grid-gutter-width / 2;
     margin-bottom: $grid-gutter-width / 2;
     background: $white;
+    border-bottom: 1px solid #dfdfdf;
```
Comment (NeOMakinG) [inline admin-dev/themes/new-theme/scss/pages/product/product_page.scss]:
```suggestion
    border-bottom: 1px solid $color-separator;
```

image

File: admin-dev/themes/new-theme/scss/pages/stock/stock_page.scss [hunk 5]
```diff
@@ -5,14 +5,26 @@
...
   padding: 0;
 
   .header-toolbar {
     z-index: 989;
+    display: block;
     padding-left: 0;
     margin: -($grid-gutter-width / 2);
+    margin-top: -1rem;
     margin-bottom: 1rem;
+    border-bottom: 0.0625rem solid #dfdfdf;
```
Comment (NeOMakinG) [inline admin-dev/themes/new-theme/scss/pages/stock/stock_page.scss]:
```suggestion
    border-bottom: 0.0625rem solid $color-separator;
```

image

File: admin-dev/themes/new-theme/scss/pages/product/product_page.scss (hunk 4 above)
Comment (NeOMakinG) [reply admin-dev/themes/new-theme/scss/pages/product/product_page.scss]: Never use hard coded colors, it hardens theming the project (for the future dark mode for example 😄 )

**Review by NeOMakinG — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by hibatallahAouadni** [comment]
Hello @mparvazi 

Thanks for your PR!
The stock page seems good :heavy_check_mark: 

image

- Product edit standard page :heavy_check_mark: (only the shadow is displayed)
image

- Product edit experimental page :heavy_check_mark: (only the shadow is displayed)
image

LGTM, QA :heavy_check_mark: 

Thanks!

**Comment by Progi1984** [comment]
Thanks @mparvazi & @hibatallahAouadni

**Pull request merged**